recipes/BEHST/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+echo "Running build.sh"
+
+echo $(pwd)
+echo $(ls)
+
+cd ./bin/
+
+mkdir ../results/
+
+mkdir ../temp/
+
+
+sh project.sh ../data/pressto_LUNG_enhancers.bed DEFAULT_EQ DEFAULT_ET
```

Comment:
The previous edit did not work.
I changed to this:

PREFIX=.
```

cd $PREFIX/bin/

mkdir $PREFIX/../results/

mkdir $PREFIX/../temp/
```

The recipe passed the tests on Travis.ci. I hope it's fine now.